scripts/ci/release/comment_pr_trigger_phrases.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# This script adds comments from a txt file, in the given GitHub Pull Request.
+# test/resources/mass_comment.txt file has the Trigger Phrases for running Gradle Build and PostCommit/PreCommit tests.
+
+file="./test/resources/mass_comment.txt"
+GITHUB_PR=$1
+while IFS= read -r trigger_phrase
+do
+    gh pr comment "$GITHUB_PR" --body "$trigger_phrase"
```

Comment:
Totally, that's why I was thinking on having for the moment the `jenkins_trigger_phrases.txt` which includes the trigger phrases of the Jenkins Jobs that won't be migrated to GitHub Actions in this first iteration/migration. 

On the other hand, if you think commenting is out of scope and it would be better to not have it yet, I can remove the `Comment Jenkins Trigger Phrases in PR` step of the workflow.

Thank you!

Reply:
I think leaving it in is the right thing. My point is just that once we've fully migrated to GHA, AFAIK we won't have trigger phrases available to us and that functionality will need to be replaced in this process.

Reply:
Yes, I agree. I think it would be great to start thinking of some GHA triggers out of the Continuous Integration ones. 
Thanks Danny for your comments!